domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/casetype/CaseTypeService.java

```diff
@@ -12,6 +12,8 @@
 
     void createAll(JurisdictionEntity jurisdiction, Collection<CaseTypeEntity> caseTypes);
 
+    Optional<Boolean> findCaseTypeExists(String reference, String jurisdictionId);
```

Comment:
can we call this method `caseTypeExists`? why are not doing a find here. A query is either an exist or a find.

Reply:
done